src/ds/d400/d400-factory.cpp

```diff
@@ -670,6 +672,9 @@ namespace librealsense
                   dev_info, d400_device::_hw_monitor, get_firmware_logs_command(), get_flash_logs_command() )
         {
             check_and_restore_rgb_stream_extrinsic();
+            if( _fw_version >= firmware_version( 5, 15, 1, 224 ) )
```

Comment:
Is this the formal FW version or just a temporary build?

Reply:
This should change to 5.16.0.0 before we merge it